File: 9-palindrome-number/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        long long sum = 0;
        int temp = x;
        while (x>0) {
            sum = sum * 10 + (x % 10);
            x /= 10;
        }
        if (temp == sum){
            return true;
        }
        else
            return false;
    }
};

File: 9-palindrome-number/palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        rev = 0
        temp = x
        while x>0:
            rev = rev * 10 + (x % 10)
            x //= 10
        if temp == rev:
            return True
        else:
            return False
                
        